lib/rdoc/code_object.rb

```diff
@@ -96,6 +96,11 @@ class RDoc::CodeObject
 
   attr_accessor :viewer
 
+  ##
+  # When mixed-in to a class, this points to the Context in which it was originally defined.
+
+  attr_accessor :mixin_from
```

Comment:
Let's follow the convention of adding it to `initialize`?

Reply:
Done.

Reply:
Done.